### ISSUE TITLE
MVP - Add API Data to the DOM

### ISSUE THREAD

**Issue by PrimalOrB** [PrimalOrB/find_your_drink#7]
- [x] Cocktails
- [x] Beers
- [x] Historical Facts

**Comment by Mantablast** [comment]
I have discovered that the cocktail API encloses 15 measurement options (spaces) and 15 ingredient spaces.  Most  cocktails in the API  barely have 15 ingredients.  It would be ideal to also create an if function during this task to avoid creating a text element when the value of the key is "null".  For example..

`if (strIngredient5 === "null" {
//trim function
//$remove class or div
//$remove element?  
`
Basically, if we were to keep these null elements on the page, it probably would take up space and not look very good.  When the html goes live and we start attaching elements we can look at fixes for this.
Cheers

**Comment by PrimalOrB** [comment]
A good method is to make an array of the ingredients. Then, ok the HTML make a blank UL element. 

In the javascript either loop over the array, or use jquery each, go generate and append LI elements to the DOM. This way it only makes entries that exist
